src/System.Runtime/tests/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Tests
+{
+    public static class FormatterServicesTests
+    {
+    	private static Func<Type, Object> geo = FormatterServices.GetUninitializedObject;
+
+        [Fact]
+        public static void NoArgument()
+        {
+            Assert.Throws<ArgumentNullException>(() => geo(null));
+        }
+
+        [Theory]
+        [InlineData(typeof(String))]
+        [InlineData(typeof(int*))]
+    	public static void Throws_ArgumentException(Type type)
+    	{
+            Assert.Throws<ArgumentException>(() => geo(type));
+    	}
+
+    	[Theory]
+    	[InlineData(typeof(Array))]
+        [InlineData(typeof(ICollection))]
+        [InlineData(typeof(Stream))]
+        public static void Throws_MemberAccessException(Type type)
```

Comment:
I'd find it helpful if the name of the test described why the exception was being thrown / what the commonality is between the types, e.g.
```InterfacesAndAbstractClasses_Throws_MemberAccessException```
```ClassWithDefaultConstructor_Success```
etc.